graylog2-server/src/test/java/org/graylog/testing/mongodb/MongoDBExtensionTest.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog.testing.mongodb;
+
+import com.mongodb.client.MongoCollection;
+import org.bson.Document;
+import org.bson.conversions.Bson;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import static com.mongodb.client.model.Filters.eq;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class MongoDBExtensionTest {
+    @Nested
+    class ProgrammaticInstanceRegistration {
+        @SuppressWarnings("unused")
+        @RegisterExtension
+        MongoDBExtension mongodbExtension = MongoDBExtension.create();
+
+        @Test
+        @MongoDBFixtures("MongoDBExtensionTest-1.json")
+        void test(MongoDBTestService mongodb) {
```

Comment:
I think it makes sense to give all tests meaningful names. I would call this one `importsCollectionFromFixture` or something similar.

Reply:
:+1: